doc/Customization-guide/Endpoint-SDKs/C-SDK/Porting-guide/index.md

```diff
@@ -4,3 +4,69 @@ title: Porting guide
 permalink: /:path/
 sort_idx: 40
 ---
+
+{% include variables.md %}
+
+* TOC
+{:toc}
+
+
+# Prerequisites
+
+Kaa C SDK can be ported to all devices  with or without any operations sytem. For example it can be ported to: Linux, Windows, Android, IOS, FireFox, Solaris, microcontrollers with or without any Rtos, etc.
```

Comment:
We also support 64-bit ones.

We also don't require POSIX-compliance. ESP8266 is no way posix-compliant, still we support it.